src/helpers/eip1559.ts

```diff
@@ -0,0 +1,13 @@
+import { Network, StoreAccount, WalletId } from '@types';
+
+// @todo Figure out if anything else needs to be disabled?
```

Comment:
Does WalletConnect support EIP-1559? And what happens if you try to sign an EIP-1559 transaction through MetaMask with a Ledger?

Reply:
No idea and good questions. We should figure that out.